src/dagmc.cpp

```diff
@@ -804,4 +834,4 @@ int32_t next_cell(int32_t surf, int32_t curr_cell, int32_t univ);
 
 } // namespace openmc
 
-#endif // DAGMC
+#endif // DAGMC
```

Comment:
It appears this file maybe has some strange ending on it? I see this:

<img width="291" alt="image" src="https://github.com/openmc-dev/openmc/assets/18088906/296342d2-6a6a-4153-9803-99a9c7990a16">

Not completely sure what that means.

Reply:
I think this is just a warning that there's no newline at the end of the file, which I've now added.